.github/workflows/unit-tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Check Black
         run: black . --check --diff
       - name: Check Ruff
-        run: ruff . --format github
+        run: ruff . --output-format github
```

Comment:
`ruff` has a formatter built in now, which potentially removes the need for `black`.